rmgpy/util.py

```diff
@@ -176,6 +176,9 @@ def parse_command_line_arguments(command_line_args=None):
     parser.add_argument('-t', '--walltime', type=str, nargs=1, default='00:00:00:00',
                         metavar='DD:HH:MM:SS', help='set the maximum execution time')
 
+    parser.add_argument('-i', '--maxiter', type=int, nargs=1, default=None,
```

Comment:
Is this `--maxiter` or `-i` flag in the documentation?

Reply:
It shows up in the command line help documentation by typing `python rmg.py -h`. I don't think we document these flags anywhere else. If we were to add it to the documentation, I am trying to think of where it should go. Maybe a small new section in the RMG documentation outlining a few tips for profiling RMG for developers? Maybe include it instead on the GitHub wiki? Let me know what you think

Reply:
Maybe they deserve a new section, but I think the command line options should be in the main documentation.

Reply:
Sounds good. I'll add a new section underneath `1.6 Running a Job` to the main documentation that demonstrates the various input flags. I'll also create a new section with tips on both time and memory profiling RMG jobs.

Reply:
That would be great. Thank you!